packages/api-nest/src/library/library.resolver.ts

```
@@ -122,6 +123,28 @@ export class LibraryResolver {
+    description: 'Update notebook content for a library item',
+  })
+  @UseGuards(JwtAuthGuard)
+  async updateNotebook(
+    @CurrentUser() user: User,
+    @Args('id', { type: () => String, description: 'Library item ID' })
+    id: string,
+    @Args('input', {
+      type: () => UpdateNotebookInput,
+      description: 'Notebook content',
+    })
+    input: UpdateNotebookInput,
```

ESLint

[error] Delete `,`

(prettier/prettier)